src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -568,6 +568,8 @@ COUCH_STALE_QUERY = 'update_after'
 API_THROTTLE_REQUESTS = 5
 API_THROTTLE_TIMEFRAME = 10
 
+ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE = {{ localsettings.ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE | default(False) }}
```

Comment:
nit: I'd suggest

```suggestion
{% if 'ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE' in localsettings %}
ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE = {{ localsettings.ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE }}
{% endif %}
```

so that for environments on which there's no change they won't see any diff related to this PR when they next update their localsettings.

Reply:
Cool, thanks for making the change! 
(Btw I don't have the ability to merge this PR, so you'll have to do it for me)

Reply:
Yup! Was just waiting for tests